### PR TITLE
Add notes about data configuration and testing

### PR DESCRIPTION
**Pull request by brianhlin**

### CODE REVIEW

File: docs/services/topology.md [hunk 1]
```diff
@@ -83,20 +97,44 @@ All changes should be tested on the ITB instance before deploying to production.
...
         :::console
         # git checkout -b <BRANCH>
 
-1.  If the data format has changed in an incompatible way, edit `/etc/opt/topology/config-itb.py`
-    and change `TOPOLOGY_DATA_DIR` and `CONTACT_DATA_DIR` to point to a new directory so the previous data
-    does not get overwritten with incompatible data.
+1.  Verify that you are using the intended data associated with the code you are testing:
+
+    !!! note
+        If the data format has changed in an incompatible way, edit the ITB configuration file and change
```
Comment (matyasselmeci) [inline docs/services/topology.md]: You shouldn't change `CONTACT_DATA_DIR` if the contact data format didn't change, and you shouldn't change `TOPOLOGY_DATA_DIR` if the topology data format didn't change.

File: docs/services/topology.md [hunk 2]
```diff
@@ -83,20 +97,44 @@ All changes should be tested on the ITB instance before deploying to production.
 If you can, test them on your local machine first.
 These instructions assume that the code has not been merged to master.
 
-1.  Update the Git clone at `/opt/topology-itb`:
+1.  Update the ITB software installation at `/opt/topology-itb`:
 
         :::console
         # cd /opt/topology-itb
         # git fetch --all
```
Comment (matyasselmeci) [inline docs/services/topology.md]: Say something about adding your repo as a remote if necessary. Just use the instructions from 125--137. The `git checkout` command should also be changed to match 132.

File: docs/services/topology.md (hunk 1 above)
Comment (matyasselmeci) [reply docs/services/topology.md]: Also, tell people to save the old version so they can revert the changes later. (This should be common sense but common sense... isn't.)

File: docs/services/topology.md [hunk 3]
```diff
@@ -118,6 +156,8 @@ These instructions assume that the code has not been merged to master.
 
 1.  If you made config changes to `/etc/opt/topology/config-itb.py`, revert them.
 
+1.  If you checked out a different branch for data, revert it back to the old branch
```
Comment (matyasselmeci) [inline docs/services/topology.md]: end with a "."

**Comment by brianhlin** [comment]
Alternative commit message: "how deep can the nested lists go?"

**Pull request merged**